src/Popup/popup.js

```diff
@@ -8,6 +8,7 @@
 		const container = document.createElement("div");
 		container.id = "chatrix-popup-container";
 		document.body.appendChild(container);
+        container.style.zIndex = 1;
```

Comment:
@psrpinto space vs tab bit here:

```
diff --git a/src/Popup/popup.js b/src/Popup/popup.js
index fe69cd2..fb180e8 100644
--- a/src/Popup/popup.js
+++ b/src/Popup/popup.js
@@ -8,6 +8,7 @@
                const container = document.createElement("div");
                container.id = "chatrix-popup-container";
                document.body.appendChild(container);
+        container.style.zIndex = 1;

                Chatrix.renderPopup(container, {
                        defaultHomeserver: config.defaultHomeserver,
```

Leave it as is or fix it? I thought Github used to show the difference 🤔

Reply:
I guess ideally we would set up a linter for js under `src/`. Could also just fix this one instance for now, I can fix.

Reply:
https://github.com/Automattic/chatrix/pull/210